Exiled.API/Extensions/Item.cs

```diff
@@ -308,5 +308,15 @@ public static void SetOther(this Player player, Inventory.SyncItemInfo weapon, O
                 }
             }
         }
+
+        /// <summary>
+        /// Gets the default durability of any item.
+        /// </summary>
+        /// <param name="item">The <see cref="ItemType">item</see> that you want to get durability of.</param>
+        /// <returns>Returns the item durability.</returns>
+        public static float GetDefaultDurability(ItemType item)
+        {
+            return Pickup.Inv.GetItemByID(item).durability;
```

Comment:
Make it not throw an exception when `Pickup.Inv.GetItemByID(item)` returned null